packages/core/aqua-wrapper/src/generate/header.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default function(isJs: boolean, aquaVersion: string) {
+    return `/**
+ *
+ * This file is auto-generated. Do not edit manually: changes may be erased.
+ * Generated by Aqua compiler: https://github.com/fluencelabs/aqua/.
+ * If you find any bugs, please write an issue on GitHub: https://github.com/fluencelabs/aqua/issues
+ * Aqua version: ${aquaVersion}
+ *
+ */
```

Comment:
since `getAquaApiVersion` is used only in one place and does only property access and nothing more - I would suggest removing it together with removing the need to pass aquaVersion as an argument here. Also the text seems repetitive and doesn't contain the version of the tool that actually produced the file. Text can be improved to look something like this:
```suggestion
import pkg from '../../package.json' assert { type: 'json' };

export default function(isJs: boolean) {
    return `/**
 * This file is generated using @fluencelabs/aqua-wrapper@${pkg['version']} with @fluencelabs/aqua-api@${pkg.dependencies['@fluencelabs/aqua-api']}
 * If you find bugs in generated AIR, please create an issue at https://github.com/fluencelabs/aqua/issues
 * If you find bugs in generated JS/TS, please create an issues at https://github.com/fluencelabs/js-client/issues
 */
```
Also I think there is absolutely no need for this file to exist. I would rather put all this right inside packages/core/aqua-wrapper/src/generate/index.ts - this function is extremely simple, used only in one place and does only string enterpolation and some ternary depending on isJs - you already do this in packages/core/aqua-wrapper/src/generate/index.ts. So I believe less nesting and files would be simpler, while still being as readable (or even more so, cause you will be able to see everything in one place without going to a different file)

Reply:
I cannot use
`import pkg from '../../package.json' assert { type: 'json' };`
because tsc will add that file entirely to dist folder and we would have package.json duplicate which is strange.

Reply:
I wouldn't concern too much about this stuff. This is not intended for web so it's not as scarry to have some extra 488 bytes.
If you have build steps you could also read and replace during the build if you wanna save some bytes

Reply:
It's not about bytes, just keeping 2 same files in the dist folder seems weird to me.

Reply:
With a build step you won't have two files, if you want to avoid that